content/reference/api/http.md

```diff
@@ -2189,8 +2344,8 @@ On success, the call to this endpoint will return with 200 and the following bod
         {
             "Type": "<int>"
             "Path": "<string>"
-            "Before": { "/": "<cid-string>" }
-            "After": { "/": "<cid-string>" }
+            "Before": "<string>"
```

Comment:
It seems we have started using `{ "/": <cid>}` like paths here and the generator does not know.

Reply:
solved this and other small issues